src/providers/Bee.tsx

```diff
@@ -113,11 +112,13 @@ function getStatus(
     blockchainConnection: Boolean(nodeAddresses?.ethereum),
     debugApiConnection: Boolean(debugApiHealth?.status === 'ok'),
     apiConnection: apiHealth,
-    topology: Boolean(topology?.connected && topology?.connected > 0),
+    // FIXME: `REACT_APP_DEV_MODE` is a temporary workaround to be able to develop with only one node
```

Comment:
Would you mind adding this to the readme? That also increases the chance we don't forget to remove it :)